.github/dependabot.yml

```diff
@@ -1,21 +1,16 @@
 version: 2
 updates:
+  # Weekly major/minor/patch version updates
   - package-ecosystem: 'npm'
     directory: '/'
     schedule:
-      interval: 'daily'
+      interval: 'weekly'
     groups:
       astro:
         patterns:
           - '@astrojs/*'
           - 'astro*'
       prod-dependencies:
         dependency-type: 'production'
-        update-types:
-          - 'minor'
-          - 'patch'
       dev-dependencies:
         dependency-type: 'development'
-        update-types:
-          - 'minor'
-          - 'patch'
```

Comment:
@carlosallexandre 
i can setup a PR if you like but actually what i can propose is this one

```yaml
version: 2
updates:
  # Weekly major/minor/patch version updates
  - package-ecosystem: 'npm'
    directory: '/'
    schedule:
      interval: 'weekly'
    ignore:
      - dependency-name: "*"
        update-types: [ "version-update:semver-major" ]
    groups:
      astro:
        patterns:
          - '@astrojs/*'
          - 'astro*'
      prod-dependencies:
        dependency-type: 'production'
        update-types:
          - "minor"
          - "patch"
      dev-dependencies:
        dependency-type: 'development'
        update-types:
          - "minor"
          - "patch"

  - package-ecosystem: "npm"
    directory: "/"
    groups:
      prod-dependencies:
        dependency-type: "production"
        update-types:
          - "major"
      dev-dependencies:
        dependency-type: "development"
        update-types:
          - "major"
    ignore:
      - dependency-name: "*"
        update-types: [ 
          "version-update:semver-minor",
          "version-update:semver-patch"
        ]
    schedule:
      interval: "monthly"

```

Reply:
This gaves that:

> Update configs must have a unique combination of `package-ecosystem`, `directory`, and `target-branch`. Ecosystem `npm` has overlapping directories.

Reply:
So we can add a target branch to major

Reply:
Done...

Thanks @XaaXaaX